From 0d423732d83f3129abfe6c7a5f4d0f8bfb569cb4 Mon Sep 17 00:00:00 2001
From: Ming Liu <liu.ming50@gmail.com>
Date: Sat, 31 Oct 2020 15:41:39 +0100
Subject: [PATCH] colibri-imx6ull: use distro boot by default

Use distro boot for raw NAND modules by default. This is required
for Mender.

Upstream-Status: Inappropriate [ mender specific ]

Signed-off-by: Ming Liu <liu.ming50@gmail.com>
---
 include/configs/colibri-imx6ull.h | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/include/configs/colibri-imx6ull.h b/include/configs/colibri-imx6ull.h
index e2c31cd4e0..c5b437c494 100644
--- a/include/configs/colibri-imx6ull.h
+++ b/include/configs/colibri-imx6ull.h
@@ -83,10 +83,11 @@
 		"ubi read ${fdt_addr_r} dtb && " \
 		"run fdt_fixup && bootz ${kernel_addr_r} - ${fdt_addr_r}\0" \
 
-#define CONFIG_BOOTCOMMAND "run ubiboot; " \
-	"setenv fdtfile " FDT_FILE " && run distro_bootcmd;"
+/* Run Distro Boot Script */
+#define CONFIG_BOOTCOMMAND "run distro_bootcmd;"
 
 #define BOOT_TARGET_DEVICES(func) \
+	func(UBIFS, ubifs, 0) \
 	func(MMC, mmc, 0) \
 	func(USB, usb, 0) \
 	func(DHCP, dhcp, na)
-- 
2.29.0

